elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -333,9 +340,13 @@ def grade(element_html, data):
 
     score = 0
     if (submitted_key is not None and submitted_key == correct_key):
-        score = 1
+        score = 1           
 
-    data['partial_scores'][name] = {'score': score, 'weight': weight}
+    for option in data['params'][name]:
+        if option['key'] == submitted_key:
+            feedback = option['feedback']
```

Comment:
One thing I'd like to bring up: in the `grade()` function, the `data['submitted_answers']` object doesn't contain the feedback I inserted into the answer tuples, so at the moment I'm not sure if there's another way to grab the feedback from the submitted answer without using a for loop.

Reply:
I believe this is a reasonable solution.